src/sentry/incidents/logic.py

```diff
@@ -946,6 +947,21 @@ def create_alert_rule_activation_condition(
     return condition
 
 
+def create_alert_rule_activation(
+    alert_rule: AlertRule,
+    metric_value: int | None = 100,
```

Comment:
Feels a little weird that we have a default here. I assume it's for tests, you should probably do the defaulting there